andela/ah-shakas-frontend
# #160577444 rating an article

## What does this PR do?
 - Ensures that a user is able to rate an article with any rate in scale of 1 - 5
## Description of Task to be completed?
- 
## How should this be manually tested?
- git clone the repository and checkout to the branch
```https://github.com/andela/ah-shakas-frontend.git```
```cd ah-shakas-frontend```
```git checkout ft-rate-articles-160577444```
```yarn install```
- Create a `.env` file with the following values
  ```
  REACT_APP_BASE_URL='https://ah-shakas-staging.herokuapp.com/api'
  REACT_APP_UPLOAD_BASE_URL=
  REACT_APP_CLOUD_NAME=
  REACT_APP_UPLOAD_PRESET=
  ```
  The `CLOUD_NAME` and the `UPLOAD_PRESET` are obtained from your cloudinary account.

```yarn start```
- navigate through the articles to see the ratings of each article. You can login to make ratings on the article

## Any background context you want to provide?
- NA
## What are the relevant pivotal tracker stories?
[#160577444](https://www.pivotaltracker.com/story/show/160577444)

## Packages installed
- axios
- react-star-ratings
- redux-mock-store

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos
- [ x] Test



## Screenshots (if appropriate)
<img width="950" alt="screen shot 2018-11-16 at 21 28 57" src="https://user-images.githubusercontent.com/39160236/48647712-d5564a80-e9fd-11e8-8c62-734ebec709d1.png">


## Questions:
<to be deleted>
